ios/RCTFBSDK/login/RCTFBSDKLoginManager.m

```diff
@@ -104,14 +103,8 @@ - (void)_loginWithPermissions:(NSStringArray *)permissions
       resolve(RCTBuildResultDictionary(result));
     }
   };
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-  if (isRead) {
-    [_loginManager logInWithReadPermissions:permissions handler:requestHandler];
-  } else {
-    [_loginManager logInWithPublishPermissions:permissions handler:requestHandler];
-  }
-#pragma clang diagnostic pop
+
+  [_loginManager logInWithPermissions:permissions fromViewController:nil handler:requestHandler];
```

Comment:
@janicduplessis does this works the same way on android too? because if it does we should remove the double methods on js side. right now we have `loginWithReadPermissions` and `loginWithPublishPermissions` on js side, but on iOS it runs into a single `loginWithPermissions` method.

Reply:
I did a bit of research and on Android the 2 different methods are not deprecated but do end up calling the same `logIn` method.

I think we could change the js api to have a single `logIn` method and deprecate the 2 other ones.

Reply:
sounds perfect. i see you already did something similar 👍